myTrip/photo/models.py

```diff
@@ -72,17 +86,21 @@ def to_dict(self):
                 {
                     'id': id,
                     'src': source link,
-                    'user_id': user id,
+                    'user': user id,
                     'trip_id': trip id,
                     'checkpoit_id': checkpoint id,
-                    'description': description text
+                    'description': description text,
+                    'created': time when created,
+                    'last updated': time when last updated
                 }
         """
         return {
             "id": self.id,
             "src": self.src,
-            "user_id": self.user_id,
-            "trip_id": self.trip_id,
-            "checkpoint_id": self.checkpoint_id,
-            "description": self.description
+            "user": self.user.id,
+            "trip_id": self.trip.id if self.trip else None,
+            "checkpoint_id": self.checkpoint.id if self.checkpoint else None,
+            "description": self.description,
+            "created": self.created_at,
+            "last updated": self.updated_at
```

Comment:
It's a bad practice to use a space in the names of the keys name.
For fields created_at and updated_at use keys with the same names